### ISSUE TITLE
Navigation in tree view not working

### ISSUE THREAD

**Issue by cary-rowen** [mltony/nvda-indent-nav#20]
Hi @mltony 

For example in NVDA's input gesture dialog, Moves to the next child/parent in tree view, does not work as expected.
NVDA reports "NO child/no parent"

Are you still maintaining the navigation in treeview? It also doesn't follow the key layout settings in indent-nav.

**Comment by mltony** [comment]
It only works in thunderbird tree views. I never found time to make it to work in other tree views. So I kind of keep it for Thunderbird only.

**Comment by cary-rowen** [comment]
If so, can we split it into separate add-ons?
For now, it takes up a lot of global command gestures, but its scope of use is quite limited.

**Comment by mltony** [comment]
Sounds good - just did as you suggested. Will update in add-on store later - when they fix submission bug.

**Comment by cary-rowen** [comment]
Great, thanks @mltony
